clio-server/src/main/scala/org/broadinstitute/clio/server/service/SearchService.scala

```diff
@@ -15,6 +16,23 @@ import org.broadinstitute.clio.transfer.model.ClioIndex
   */
 class SearchService private[server] (searchDAO: SearchDAO) {
 
+  /**
+    * Run a simple string query.
+    * @param simpleJson The simple json to query.
+    * @tparam CI The type of the TransferKey to query.
+    * @return The result of the query.
+    */
+  def simpleStringQueryMetadata[Input, CI <: ClioIndex](simpleJson: Json)(
```

Comment:
After playing with the branch, I don't think we need this method. We can reuse the existing `queryMetadata` as long as we have a `ElasticsearchQueryMapper[clioIndex.KeyType]`, and we already have all the implicits we need to build one of those.

The only change we need to make in the query-mapper is to change `flattenVals` to support non-options:
```scala
private def flattenVals(queryInput: Input): Seq[(String, _)] = {
    val vals = inputMapper.vals(queryInput).toSeq
    val flattened = vals flatMap {
      case (name, value) =>
       // Currently, this throws when encountering a non-option
        value match {
          case opt: Option[_] => opt.map(name -> _)
          case nonOpt         => Some(name -> nonOpt)
        }
    }
    flattened sortBy {
      case (name, _) => name
    }
  }
```